packages/paste-core/components/card/src/utilities.tsx

```diff
@@ -0,0 +1,19 @@
+const marginProps = ['margin', 'marginTop', 'marginRight', 'marginBottom', 'marginLeft'];
+const paddingProps = ['padding', 'paddingTop', 'paddingRight', 'paddingBottom', 'paddingLeft'];
```

Comment:
For clarity do you mind just removing the ones we're not actually using anywhere?